resources/views/home/modal/show-reviews.blade.php

```diff
@@ -12,13 +12,34 @@
                 <div class="modal-body" style="font-family: EB Garamond">
                     <div class="align-items-center justify-content-center flex-row d-flex">
                         {{--  レビューの星　--}}
-                        <div style="width: 320px; height: 180px; border: 1px solid #0C2C04" class="rounded d-flex flex-column justify-content-center align-items-center ms-5">
-                            <div class="mb-2">
-                                <i class="fa-solid fa-star fa-3x"></i>
-                                <span class="h1 ms-2">{{ number_format($averageRating, 1) }}</span>
+                        <div style="width: 320px; height: 180px; border: 1px solid #0C2C04" class="review-star-container rounded d-flex flex-column justify-content-center align-items-center ms-5">
+                            @php
+                            $averageRating = number_format($averageRating, 1);
+                            $fullStars = floor($averageRating);
+                            $halfStar = ($averageRating > $fullStars) ? true : false;
```

Comment:
`$halfStar = ($averageRating > $fullStars);` 
is enough because this will return true or false